spec/models/generic_work_spec.rb

```diff
@@ -37,17 +37,36 @@
     let(:work) { described_class.new(state: state) }
     subject { work.suppressed? }
 
-    context "when the state is 'pending'" do
-      let(:state) { 'pending' }
+    context "when it has its initial state" do
+      let(:state) { Vocab::FedoraResourceStatus.inactive }
       it { is_expected.to be true }
     end
 
     context "when the state is something else" do
-      let(:state) { ::RDF::URI('http://fedora.info/definitions/1/0/access/ObjState#active') }
+      let(:state) { Vocab::FedoraResourceStatus.active }
       it { is_expected.to be false }
     end
   end
 
+  context "when activating or deactivating a work" do
+    let(:work) { described_class.new }
+    describe "#activate" do
+      subject { work.activate }
+      it "sets its state to active" do
+        subject
```

Comment:
Not sure what line 56 is adding. Can you rework this test to be more succinct? Like

```ruby
describe '#activate' do
  subject { work.activate }
  it { is_expected.not_to be_suppressed }
end
```